storage/rocksdb/ha_rocksdb.cc

```diff
@@ -417,6 +433,8 @@ static char *rocksdb_strict_collation_exceptions;
 static my_bool rocksdb_collect_sst_properties = 1;
 static my_bool rocksdb_force_flush_memtable_now_var = 0;
 static my_bool rocksdb_force_flush_memtable_and_lzero_now_var = 0;
+static my_bool rocksdb_enable_ttl = 1;
```

Comment:
Actually, these are all GLOBAL sys_vars, they should not need initialization at all as the defaults should be set via the MYSQL_SYSVAR_* macros and init process, no?